java/server/src/org/openqa/selenium/grid/distributor/local/LocalDistributor.java

```diff
@@ -207,11 +210,17 @@ public CreateSessionResponse newSession(HttpRequest request)
       return sessionResponse;
     } catch (SessionNotCreatedException e) {
       span.setAttribute("error", true);
-      span.setStatus(Status.ABORTED.withDescription(e.getMessage()));
+      span.setStatus(Status.ABORTED);
+      Map<String, EventAttributeValue> attributeValueMap = new HashMap<>();
+      attributeValueMap.put("Error Message", EventAttribute.setValue(e.getMessage()));
+      span.addEvent("Session not created",attributeValueMap);
       throw e;
     } catch (IOException e) {
       span.setAttribute("error", true);
-      span.setStatus(Status.UNKNOWN.withDescription(e.getMessage()));
+      span.setStatus(Status.UNKNOWN);
+      Map<String, EventAttributeValue> attributeValueMap = new HashMap<>();
+      attributeValueMap.put("Error Message", EventAttribute.setValue(e.getMessage()));
+      span.addEvent("Unknown error in LocalDistributor while creating session",attributeValueMap);
```

Comment:
nit: make sure there are spaces after a `,`
```suggestion
      span.addEvent("Unknown error in LocalDistributor while creating session", attributeValueMap);
```

Reply:
Thank you David. I formatted the changes as needed. Sorry missed this out.